internal/resolve/catalog.go

```diff
@@ -76,6 +86,16 @@ func (r *CatalogResolver) Resolve(ctx context.Context, ext *ocv1.ClusterExtensio
 			return fmt.Errorf("error getting package %q from catalog %q: %w", packageName, cat.Name, err)
 		}
 
+		cs := catStat{CatalogName: cat.Name}
+		catStats = append(catStats, &cs)
+
+		if isFBCEmpty(packageFBC) {
+			return nil
+		}
```

Comment:
This early exit did not exist before. Are we worried about a change in behavior?

Reply:
Before, if `len(packageFBC.Bundles)` was 0, we would evaluate/run the predicates against a 0-length slice, which would return a 0-length slice, and then we'd `return nil` in line 103 (old) / line 124 (new)

So I don't think there's a behavior change here other than skipping some unnecessary predicate setup.